src/CMakeLists.txt

```diff
@@ -69,7 +75,7 @@ if(ANDROID)
   set(FAST_TOKENIZER_FROM_SOURCES ON)
 else()
   set(FAST_TOKENIZERS_SUPPORTED ON)
-  if(USE_ABI0)
+  if(USE_ABI0 OR NOT build_type MATCHES Release)
```

Comment:
or `build_type STREQUAL Debug` ?
I suppose MinRelSize or RelWithDebInfo shoul work with prebuilt tokenizers?

Reply:
I suppose in case of MSVC (multi generator config), we will always build FastTokenizers from sources, right?

Reply:
set CMAKE_BUILD_TYPE to Release by default for multiconfig generators and do not enable fast_tokenizers by default